ZUGFeRD/InvoiceDescriptor22UblReader.cs

```diff
@@ -621,7 +629,26 @@ private static TradeLineItem _parseTradeLineItem(XmlNode tradeLineItem, XmlNames
             //  item._AdditionalReferencedDocuments.Add(_readAdditionalReferencedDocument(referenceNode, nsmgr));
             //}
 
-            return item;
+            //Add main item to result list
+            resultList.Add(item);
+
+            //Find sub invoice lines recursively
+            //Note that selectnodes also select the sub invoice line from other nodes
+            XmlNodeList subInvoiceLineNodes = tradeLineItem.SelectNodes(".//cac:SubInvoiceLine", nsmgr);
+            foreach (XmlNode subInvoiceLineNode in subInvoiceLineNodes)
+            {
+                List<TradeLineItem> parseResultList = _parseTradeLineItem(subInvoiceLineNode, nsmgr, item.AssociatedDocument.LineID);
+                foreach(TradeLineItem resultItem in  parseResultList)
```

Comment:
please remove one of the two spaces after in (feel free to choose which one :) )